src/categorical_algebra/DPO.jl

```diff
@@ -12,15 +12,19 @@ Apply a rewrite rule (given as a span, L<-I->R) to a ACSet
 using a match morphism `m` which indicates where to apply
 the rewrite.
 """
-function rewrite_match(L::ACSetTransformation, R::ACSetTransformation,
-                       m::ACSetTransformation)::ACSet
+function rewrite_match_maps(L::ACSetTransformation, R::ACSetTransformation,
+                         m::ACSetTransformation)::Vector{ACSetTransformation}
   dom(L) == dom(R) || error("Rewriting where L, R do not share domain")
   codom(L) == dom(m) || error("Rewriting where L does not compose with m")
-  (k, _) = pushout_complement(L, m)
-  l1, _ = pushout(R, k)
-  return codom(l1)
+  (ik, kg) = pushout_complement(L, m)
+  rh, kh = pushout(R, ik)
+  return [ik, kg, rh, kh]
 end
 
+rewrite_match(L::ACSetTransformation, R::ACSetTransformation,
```

Comment:
It looks like the main function `rewrite_match` no longer has a docstring and that the docstring now attached to `rewrite_match_maps` applies to the old function. All exported functions should have a docstring.